tmt/steps/provision/mrack.py

```diff
@@ -776,12 +776,11 @@ def _translate_tmt_hw(self, hw: tmt.hardware.Hardware) -> dict[str, Any]:
             """ Return hw requirements from given hw dictionary """
 
             assert hw.constraint
-
-            transformed = MrackHWAndGroup(
-                children=[
-                    constraint_to_beaker_filter(constraint, logger)
-                    for constraint in hw.constraint.variant()
-                    ])
+            # Beaker,unlike instance-type-based infrastructures like AWS, does
```

Comment:
nitpick - missing space after the comma:

```
Beaker, unlike ....
```

Reply:
Thanks for your review:) 
Updated:https://github.com/teemtee/tmt/pull/3327/commits/53e7a2f3596810573221342af37aa18ef0f2d76f